tests/console-backend-service/OWNERS

```diff
@@ -0,0 +1,9 @@
+reviewers:
+  - console-reviewers
+  - m00g3n
```

Comment:
Marcin (`m00g3n`) is out atm but I'm sure he is sharing the same point of view that Mateusz and Michal so please remove him as well, and `magicmatatjahu` too